packages/common/src/hooks/chats/useChatBlastAudienceContent.ts

```diff
@@ -0,0 +1,99 @@
+import { useMemo } from 'react'
+
+import { ChatBlast, ChatBlastAudience } from '@audius/sdk'
+
+import {
+  useGetCurrentUser,
+  useGetCurrentUserId,
+  useGetPlaylistById,
+  useGetPurchasersCount,
+  useGetRemixersCount,
+  useGetTrackById
+} from '~/api'
+import { getChat } from '~/store/pages/chat/selectors'
+import { decodeHashId, getChatBlastTitle } from '~/utils'
+
+import { useProxySelector } from '../useProxySelector'
+
+export const useChatBlastAudienceContent = ({ chatId }: { chatId: string }) => {
+  const chat = useProxySelector((state) => getChat(state, chatId), [chatId])
+  const {
+    audience,
+    audience_content_id: audienceContentId,
+    audience_content_type: audienceContentType
+  } = chat as ChatBlast
+
+  const decodedContentId = audienceContentId
+    ? decodeHashId(audienceContentId) ?? undefined
```

Comment:
this is also weird - `decodeHashId` could return `null` but i want `decodedContentId` to only be `string | undefined`

Reply:
ah nice.. i will see if we can do this.